### PR TITLE
Work

### PR DESCRIPTION
**Pull request by Sampkipa**
practice  discussion code.

### CODE REVIEW

File: 3.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ fix #5
 resolves #4 #3
 
 This is a test for practice Pull requests & Merge pull requests.
+This is a test for discussion.
```
Comment (XMLCRP) [inline 3.txt:8]: >It‘s new code ,right?

**Review by XMLCRP — COMMENTED** [review]
For Sampkipa #8

File: 3.txt (hunk 1 above)
Comment (Sampkipa) [reply 3.txt:8]: Yes. I wirte some news.

Comment (XMLCRP) [reply 3.txt:8]: OK,I think it's good. I will merge it in master.

**Pull request merged**